ocean_provider/utils/accounts.py

```diff
@@ -4,15 +4,17 @@
 #
 import logging
 
-import eth_keys
 from eth_account.account import Account
 from eth_account.messages import encode_defunct
+from eth_keys import KeyAPI
+from eth_keys.backends import NativeECCBackend
 from ocean_provider.exceptions import InvalidSignatureError
 from ocean_provider.user_nonce import get_nonce
-from ocean_provider.utils.basics import get_web3
+from ocean_provider.utils.basics import get_web3, get_provider_wallet
```

Comment:
Please remove `get_provider_wallet`. It is not used.

Please make sure you have the pre-commit checks installed before committing.  They will fix errors like this automatically.

```python
pre-commit install
```